cmd/kubeadm/app/util/staticpod/utils.go

```diff
@@ -56,6 +56,7 @@ var (
 
 // ComponentPod returns a Pod object from the container, volume and annotations specifications
 func ComponentPod(container v1.Container, volumes map[string]v1.Volume, annotations map[string]string) v1.Pod {
+	priority := int32(2000001000)
```

Comment:
@neolit123 something like that ?
```suggestion
        // priority value for system-node-critical class
	priority := int32(2000001000)
```